### DISCUSSION TITLE
How can I load data from sdf file with python？

### DISCUSSION THREAD

**Discussion by jcyu96** [epochpic/epoch#238] [Q&A] [answered]
Dear all,
I‘m trying to load sdf data with python, but I have no idea exactly what the operation is, can you guys help me? 
Thanks!

**Comment by TomGoffrey** [comment, marked as answer]
Hi @jcyu96,

I just answered a similar question in #240 - could you try those instructions and let me know if you have further questions?

**Reply by yuedongning** [reply]
I am trying to write a python code to read SDF files. It will take a period of time.

**Reply by TomGoffrey** [reply]
EPOCH includes a reader for the SDF file for python. Why not use that?
